mlflow/deployments/cli.py

```diff
@@ -495,6 +496,9 @@ def validate_config_path(_ctx, _param, value):
     help="The number of workers.",
 )
 def start_server(config_path: str, host: str, port: str, workers: int):
+    if is_windows():
+        raise RuntimeError("MLflow Deployments Server does not support Windows.")
```

Comment:
```suggestion
        raise click.ClickException("MLflow Deployments Server does not support Windows.")
```

can we use `click.ClickException`?

Reply:
ooh didn't know that feature, updated!